main.pm

```diff
@@ -535,7 +539,7 @@ sub load_x11tests(){
     }
     if (gnomestep_is_applicable) {
         loadtest "x11/nautilus.pm" unless get_var("LIVECD");
-        loadtest "x11/evolution.pm" unless (check_var("FLAVOR", "Server-DVD"));
+        loadtest "x11/evolution.pm" unless (is_server);
```

Comment:
just for the record: if clauses at the end of the statement don't need () - as in the nautilus line. I'll merge it nonetheless